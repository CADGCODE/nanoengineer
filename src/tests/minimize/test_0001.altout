======= test_0001.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
== structure comparison ==
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0001.mmp  > test_0001.xyz

Total Ke = 9.315000e-20
final RMS gradient=0.994508 after 146 iterations
== structure comparison ==
at: translate (0.000000 0.000000 0.000000)
       rotate (0.000000 0.000000 0.000000)
        scale 1.000000,
standard deviation = 0.000000e+00, max delta = 0.000000e+00
======= test_0001.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0001.mmp
# Output File: test_0001.xyz
# Trace File: test_0001.trc
# Number of Atoms: 15
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#

# rms force, high force
1 2294.87 4213.83
2 1909.02 3373.96
3 1479.26 2766.58
4 1185.25 2178.78
5 1082.10 2024.80
6 831.52 1564.34
7 747.90 1478.71
8 668.17 1274.31
9 586.55 1156.59
10 512.92 1009.89
11 455.74 917.68
12 406.88 761.60
13 364.57 746.35
14 360.23 804.77
15 302.12 613.22
16 278.98 607.89
17 222.29 456.94
18 203.06 399.15
19 183.85 374.49
20 163.53 322.91
21 142.80 294.70
22 125.94 244.45
23 113.28 234.32
24 104.74 210.48
25 90.53 185.46
26 82.88 160.34
27 75.21 151.31
28 67.35 126.81
29 60.51 120.24
30 54.85 102.10
31 50.15 96.96
32 46.36 88.13
# Switching to Conjugate-Gradient
33 22.32 0.00
34 17.98 0.00
35 17.95 0.00
36 17.77 0.00
37 13.98 0.00
38 13.97 0.00
39 13.96 0.00
40 13.95 0.00
41 13.94 0.00
42 13.92 0.00
43 13.91 0.00
44 13.90 0.00
45 13.89 0.00
46 13.88 0.00
47 13.87 0.00
48 13.85 0.00
49 13.84 0.00
50 13.83 0.00
51 13.81 0.00
52 13.79 0.00
53 13.76 0.00
54 13.72 0.00
55 13.66 0.00
56 13.54 0.00
57 13.33 0.00
58 13.01 0.00
59 12.55 0.00
60 12.04 0.00
61 11.55 0.00
62 11.01 0.00
63 10.49 0.00
64 9.78 0.00
65 9.14 0.00
66 8.10 0.00
67 7.95 0.00
68 7.94 0.00
69 7.88 0.00
70 7.78 0.00
71 7.74 0.00
72 7.59 0.00
73 7.33 0.00
74 7.10 0.00
75 6.81 0.00
76 6.51 0.00
77 6.36 0.00
78 6.12 0.00
79 6.05 0.00
80 5.92 0.00
81 5.79 0.00
82 5.73 0.00
83 5.57 0.00
84 5.43 0.00
85 5.18 0.00
86 5.12 0.00
87 4.97 0.00
88 4.91 0.00
89 4.84 0.00
90 4.81 0.00
91 4.75 0.00
92 4.67 0.00
93 4.56 0.00
94 4.38 0.00
95 4.25 0.00
96 4.05 0.00
97 3.95 0.00
98 3.82 0.00
99 3.75 0.00
100 3.65 0.00
101 3.54 0.00
102 3.45 0.00
103 3.28 0.00
104 3.19 0.00
105 3.03 0.00
106 3.01 0.00
107 2.93 0.00
108 2.87 0.00
109 2.84 0.00
110 2.82 0.00
111 2.79 0.00
112 2.74 0.00
113 2.66 0.00
114 2.57 0.00
115 2.45 0.00
116 2.35 0.00
117 2.26 0.00
118 2.20 0.00
119 2.16 0.00
120 2.10 0.00
121 2.06 0.00
122 2.00 0.00
123 1.94 0.00
124 1.86 0.00
125 1.79 0.00
126 1.73 0.00
127 1.67 0.00
128 1.63 0.00
129 1.60 0.00
130 1.56 0.00
131 1.53 0.00
132 1.48 0.00
133 1.44 0.00
134 1.38 0.00
135 1.33 0.00
136 1.28 0.00
137 1.24 0.00
138 1.21 0.00
139 1.18 0.00
140 1.16 0.00
141 1.14 0.00
142 1.11 0.00
143 1.07 0.00
144 1.04 0.00
145 0.99 0.00
146 0.99 2.04
# Done: Minimization final rms: 0.994508, highForce: 2.036953
======= test_0001.xyz =======
15
RMS=0.994508
C -0.193641 2.900593 -0.026523
X 0.093601 3.502437 0.394872
X -0.623522 3.154064 -0.637458
C -1.079249 2.005273 0.890906
X -1.803795 1.958430 0.584626
X -1.090331 2.310792 1.617200
C 0.986482 2.029986 -0.552402
X 0.945121 1.985940 -1.338110
X 1.667645 2.347089 -0.314849
C -0.443634 0.583852 0.936816
X -0.955793 0.061908 0.643109
X -0.248030 0.411844 1.680547
C 0.839719 0.603152 0.054672
X 1.466374 0.446893 0.504740
X 0.762053 0.079748 -0.528147
