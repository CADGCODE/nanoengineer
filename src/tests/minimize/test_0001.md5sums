stdout FEBEB95D47D364DB1B1D1F1C30F10AF4
stderr D41D8CD98F00B204E9800998ECF8427E
test_0001.xyz B8B1597CF08E157FDFAECE8F853B97A2
