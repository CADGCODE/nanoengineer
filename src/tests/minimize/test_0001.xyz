15
RMS=0.899488
C -0.172441 2.887627 0.001225
X 0.148044 3.456804 0.443776
X -0.604323 3.184037 -0.588822
C -1.059412 1.999023 0.882326
X -1.782928 1.968888 0.571429
X -1.066867 2.288985 1.615068
C 0.941991 2.008051 -0.578258
X 0.828406 1.925343 -1.353656
X 1.640055 2.336217 -0.416896
C -0.453152 0.590634 0.898388
X -0.954386 0.090067 0.553128
X -0.298518 0.384653 1.643168
C 0.854042 0.636062 0.099369
X 1.461958 0.550880 0.591973
X 0.840532 0.074728 -0.452218
