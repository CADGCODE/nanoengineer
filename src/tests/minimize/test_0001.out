======= test_0001.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
== structure comparison ==
